lib/cisco_node_utils/cmd_ref/vrf.yaml

```diff
@@ -41,6 +41,25 @@ destroy:
   ios_xr:
     set_value: 'no vrf <vrf>'
 
+mhost_ipv4:
```

Comment:
Any number of CLI commands can share yaml entries.  It's just a question of "does it make sense".